### PR TITLE
For Technical Review

### PR DESCRIPTION
**Pull request by LukeHauck**
Hi guys,
Can you please review the SKOS optimised format for seismic-dimensionality vocab and approve if okay.
Cheers, Luke

### CODE REVIEW

File: vocabularies/seismic-dimensionality.ttl [hunk 1]
```diff
@@ -1,19 +1,53 @@
- @prefix : <http://linked.data.gov.au/def/seismic-sampling-method> .
- @prefix ssmethod: <> .
+@prefix : <http://linked.data.gov.au/def/seismic-dimensionality> .
```
Comment (KellyVance) [inline vocabularies/seismic-dimensionality.ttl]: <http://linked.data.gov.au/def/seismic-dimensionality/> .

add trailing slash

File: vocabularies/seismic-dimensionality.ttl [hunk 2]
```diff
@@ -1,19 +1,53 @@
...
- @prefix ssmethod: <> .
+@prefix : <http://linked.data.gov.au/def/seismic-dimensionality> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix ssmethod: <file:///tmp/skosify-cgiMwa8Mb/input.ttl> .
```
Comment (KellyVance) [inline vocabularies/seismic-dimensionality.ttl]: delete line for ssmethod

File: vocabularies/seismic-dimensionality.ttl [hunk 3]
```diff
@@ -1,19 +1,53 @@
...
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
```
Comment (KellyVance) [inline vocabularies/seismic-dimensionality.ttl]: <http://linked.data.gov.au/def/seismic-dimensionality> a owl:Ontology .

add ontology line back in with < delimiters

File: vocabularies/seismic-dimensionality.ttl [hunk 4]
```diff
@@ -1,19 +1,53 @@
...
- @prefix dc: <http://purl.org/dc/elements/1.1/> .
- @prefix dct: <http://purl.org/dc/terms/> .
- @prefix owl: <http://www.w3.org/2002/07/owl#> .
- @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
- @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
- @prefix sdo: <https://schema.org/> .
- @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
- @prefix xml: <http://www.w3.org/XML/1998/namespace> .
- @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+<http://linked.data.gov.au/def/seismic-dimensionality> a skos:ConceptScheme ;
```
Comment (KellyVance) [inline vocabularies/seismic-dimensionality.ttl]: :conceptScheme a skos:ConceptScheme
    dct:created "2019-09-24"^^xsd:date ;
    dct:modified "2019-09-24"^^xsd:date ;
    dct:creator "Geological Survey of Queensland" ;

change concept scheme lines and add temporal lines (change dates if known/needed)

File: vocabularies/seismic-dimensionality.ttl [hunk 5]
```diff
@@ -1,19 +1,53 @@
...
+        "4D Survey"@en,
+        "Time Lapse"@en,
+        "Time Lapse Survey"@en,
+        "Time-Lapse Survey"@en ;
+    skos:definition "The Fundamental Dataset of a 4D seismic survey is the 'Temporal Volume'; The Temporal Volume is a collection of multiple discrete recordings of a volume of the subsurface measured at successive time intervals. It is described by a function of four parameters: 1) In-Line, 2) Cross-Line, 3) Two-Way Time and 4) Repeat Index. The In-Lines and Cross-Lines intersect with each other."@en ;
+    skos:inScheme : ;
+    skos:prefLabel "4D Seismic"@en ;
+    skos:topConceptOf : .
+
+: a owl:Ontology ;
```
Comment (KellyVance) [inline vocabularies/seismic-dimensionality.ttl]: Can delete lines 49-52

Comment (LukeHauck) [reply vocabularies/seismic-dimensionality.ttl]: Done

File: vocabularies/seismic-dimensionality.ttl (hunk 1 above)
Comment (LukeHauck) [reply vocabularies/seismic-dimensionality.ttl]: Done

**Pull request closed without merging**